src/MPQEditor/MPQEditor.ts

```diff
@@ -74,6 +75,39 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
     return undefined;
   }
 
+  /**
+   * @brief A helper function to find unoccupied mpq file-name
+   * @returns valid file name for mpq configuration or undefined on failure
+   */
+  public static findMPQName(
+    baseMPQName: string,
+    dirPath: string
+  ): string | undefined {
+    if (baseMPQName.length === 0) {
+      throw new Error("Invalid mixed precision quantization file name");
```

Comment:
(optional) maybe add add a note about `throw` with invalid input in comment.

Reply:
Ahhh. Ok

Reply:
@seanshpark 
done.